### ISSUE TITLE
`formatter` lint rule only warns when importing `assert` package

### ISSUE THREAD

**Issue by pgimalac** [Antonboom/testifylint#170]
Here is a minimal reproducible example where the formatter rule should suggest not using `fmt.Sprintf`, but doesn't.
```go
package main

import (
	"fmt"
	"testing"

	"github.com/stretchr/testify/require"
)

func TestA(t *testing.T) {
	require.True(t, false, fmt.Sprintf("expected %v, got %v", true, false))
}
```

Adding a blank import of `github.com/stretchr/testify/assert` (or any assertion using it) will make the `formatter` rule complain.

**Comment by ccoVeille** [comment]
Thanks for reporting this. I appreciate your feedback, and the minimum reproducible code.

We will have a look.

This one is indeed a bug, if confirmed.

The fix should be easy :tm: (famous last words 😅🤣)

**Comment by pgimalac** [comment]
I took a look and I believe I found the issue, I'll validate and open a PR.

**Comment by Antonboom** [comment]
@pgimalac thanks!

Fixed.
